frontends/web/src/components/terms/bitsurance-terms.tsx

```diff
@@ -30,6 +33,14 @@ export const BitsuranceTerms = ({ onAgreedTerms }: TProps) => {
   const handleSkipDisclaimer = (e: ChangeEvent<HTMLInputElement>) => {
     setConfig({ frontend: { skipBitsuranceDisclaimer: e.target.checked } });
   };
+  const getPrivacyLink = (): string => {
+    const selectedLanguage = defaultLanguages[getSelectedIndex(defaultLanguages, i18n)];
```

Comment:
I think we should start using `i18n.resolvedLanguage ` like here https://github.com/digitalbitbox/bitbox-wallet-app/commit/bf70be385f422537b896ce6c853d425ee48f8f1e

Reply:
Nice! Updated also the one in bitsurance.tsx